module-3/back-end/src/models/project.js

```diff
@@ -0,0 +1,47 @@
+const {knex} = require('../db/knex')
+const {canFindById, canGetAll, canDeleteById} = require('./baseModel')
+
+const config = {
+    tableName: 'project'
+}
+
+const Model = (config) => {
+
+    const create = async project => {
+
+        return await knex(config.tableName).insert(project, [
+            'id',
+            'title',
+            'description',
+            'document_id',
+            'updated_at'
```

Comment:
Apart from having `updated_at` I think it's a good idea to also store `created_at` timestamp

Reply:
created_at will be added at the database level automatically at the moment of insert